### PR TITLE
Frogger

### PR DESCRIPTION
**Pull request by igkostyuk**
### Object Oriented JS Frogger

[Demo](https://igkostyuk.github.io/frontend-nanodegree-arcade-game-master/) |
[Code base](https://github.com/igkostyuk/frontend-nanodegree-arcade-game-master/tree/master/js)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/igkos/js-oop/js/app.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+const BLOCK_HEIGHT = 83;
+const BLOCK_WIDTH = 101;
+const TOP_OffSET = 20;
```
Comment (OleksiyRudenko) [inline submissions/igkos/js-oop/js/app.js]: A minor thing, please, make `TOP_OffSET` all caps.

File: submissions/igkos/js-oop/js/app.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    this.x < player.x + BLOCK_WIDTH - TOP_OffSET &&
+    this.y > player.y - BLOCK_HEIGHT + TOP_OffSET &&
+    this.y < player.y + BLOCK_HEIGHT - TOP_OffSET
+  ) {
+    this.interaction();
+  }
+};
+
+let Enemy = function(x, y, speed) {
+  this.sprite = "images/enemy-bug.png";
```
Comment (OleksiyRudenko) [inline submissions/igkos/js-oop/js/app.js]: File path is also a good candidate to become a constant.

File: submissions/igkos/js-oop/js/app.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+const RANDOM_SPEED = (min = 60, max = 200) =>
+  Math.floor(Math.random() * (max - min)) + min;
+
+let Element = function() {};
+
+Element.prototype.render = function() {
+  ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+};
+
+Element.prototype.collision = function() {
```
Comment (OleksiyRudenko) [inline submissions/igkos/js-oop/js/app.js]: By convention methods/functions are actions and therefore function ids should start with a verb, imperative mood, describing the action to take. Please, check and fix across entire code base

**Comment by igkostyuk** [comment]
Please, review

**Pull request merged**